kore/src/Kore/Variables/Target.hs

```diff
@@ -70,25 +70,19 @@ isTarget (NonTarget _) = False
 isNonTarget :: Target variable -> Bool
 isNonTarget = not . isTarget
 
-instance
-    SortedVariable variable
-    => SortedVariable (Target variable)
+instance (SyntaxVariable variable, SortedVariable variable)
+  => SortedVariable (Target variable)
```

Comment:
You cannot use the default instance here. The instance for `Target` must rename `Target` and `NonTarget` variables that have the same name.